usefulUtilities/modelJSONCleanup/README.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+Node script that can be used to help cleanup models.json
+
+# Use
+
+- Get an archive of the domain you would like
+- Extract the zip
+- Extract the models gzip file
+- Duplicate the config.json file and fill out the fields
+- Refer to the comments in the modelJSONCleanup file for more information on the fields in the config.json file.
+- Run `node modelJSONCleanup.js <"path to config file">`
+- If a path isn't provided for that argument, it will be assumed that the config.json is in the same directory as the js file
+- You will get a log of what was kept and deleted, if it doesn't add up, you can tweak the results
```

Comment:
Where does the log go?

Reply:
It just gets printed upon running.  
![image](https://user-images.githubusercontent.com/11789978/54958889-9f2d5800-4f14-11e9-80b4-5d0c2ad0df81.png)

Do you think it should be an actual file written?

Reply:
no, i think it's fine that it's printed to the console. i do think making sure the readme here is clear about where that log will go is important!